pyproject.toml

```diff
@@ -13,7 +13,8 @@ graphviz = "^0.20.1"
 networkx = "^3.2.1"
 pydantic = "^2.7.0b1"
 typing-extensions = "^4.9.0"
-hugr = "^0.5.0"
+#hugr = "^0.6.0"
+hugr = { git = "https://github.com/CQCL/hugr", branch = "ab/guppy-things", subdirectory = "hugr-py" }
```

Comment:
Waiting for `hugr 0.6.1`, as we need the fix from https://github.com/CQCL/hugr/pull/1422
We'll also need a tket2 release compatible with it